doc/api.rst

```diff
@@ -39,14 +39,14 @@ similar to that of Python's `str.format
 <https://docs.python.org/3/library/stdtypes.html#str.format>`_.
 They take *fmt* and *args* as arguments.
 
-*fmt* is a format string that contains literal text and replacement fields
-surrounded by braces ``{}``. The fields are replaced with formatted arguments
-in the resulting string. `~fmt::format_string` is a format string which can be
-implicitly constructed from a string literal or a ``constexpr`` string and is
-checked at compile time in C++20. To pass a runtime format string wrap it in
-`fmt::runtime`.
+*  *fmt* is a format string that contains literal text and replacement fields
+   surrounded by braces ``{}``. The fields are replaced with formatted arguments
+   in the resulting string. `~fmt::format_string` is a format string which can be
+   implicitly constructed from a string literal or a ``constexpr`` string and is
+   checked at compile time in C++20. To pass a runtime format string wrap it in
+   `fmt::runtime`.
 
-*args* is an argument list representing objects to be formatted.
+* *args* is an argument list representing objects to be formatted.
```

Comment:
I think it's better to keep this as separate paragraph, not as a list. Please revert.